db/migrate/20220808022100_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.date :date_of_birth
+      t.string :email
+      t.string :password
+      t.string :phone_number
+      t.boolean :admin?
```

Comment:
để kiểu int rồi dùng enum trong model em nhé

```suggestion
      t.integer :role
```